### ISSUE TITLE
Low detection performance on nuscenes Test Set

### ISSUE THREAD

**Issue by gaojunbin** [mit-han-lab/bevfusion#216]
Hi, thanks for your excellent work.

I recurrent bevfusion-det (use your released model) on the val set and get the same result as yours (mAP 68.52 and NDS 71.38). However, when I use this model inferenced on the test set, I submitted the result to EvalAI (nuscsnes leaderboard). I got the performance as below on the test set:
```
 "result": {"mAP": 0.6861332356736426, "mATE": 0.26553269309045924, "mASE": 0.24276069228904912, "mAOE": 0.37802993647241667, "mAVE": 0.2765107343879277, "mAAE": 0.1298454866946695, "NDS": 0.7137986635433691}}
```
I think there is a large drop with the result you showed in paper and github repo (mAP 70.23 and NDS 72.88). I would like to ask is the same model you used while getting the test set result? If not, what's the difference between the model in val set and test set?

Thanks for your help.

**Comment by kentang-mit** [comment]
Hi @gaojunbin,

I received your Email and I'll reply here. For the submission to test leaderboard, you need to train on train+val instead of on only training set. This is the common practice and we just follow what other papers did.

Best,
Haotian
